openquake/qa_tests_data/classical_risk/case_4/expected/loss_curves-001.csv

```diff
@@ -0,0 +1,7 @@
E+07 8.5331992E+07 8.5556048E+07 8.5747560E+07 8.5939080E+07 8.6130600E+07 8.6322112E+07 8.6513632E+07 8.6677704E+07 8.6841784E+07 8.7005856E+07 8.7169936E+07 8.7334008E+07 8.7474904E+07 8.7615800E+07 8.7756688E+07 8.7897584E+07 8.8038472E+07 8.8159736E+07 8.8281000E+07 8.8402264E+07 8.8523528E+07 8.8644792E+07 8.8749400E+07 8.8854008E+07 8.8958616E+07 8.9063216E+07 8.9167824E+07 8.9258264E+07 8.9348704E+07 8.9439144E+07 8.9529584E+07 8.9620024E+07 8.9698392E+07 8.9776752E+07 8.9855120E+07 8.9933480E+07 9.0011848E+07 9.0079896E+07 9.0147944E+07 9.0215992E+07 9.0284032E+07 9.0352080E+07 9.0411296E+07 9.0470512E+07 9.0529720E+07 9.0588936E+07 9.0648152E+07 9.0699784E+07 9.0751416E+07 9.0803048E+07 9.0854680E+07 9.0906312E+07 9.1306776E+07 9.1707232E+07 9.2107696E+07 9.2508160E+07 9.2908624E+07,2.3945072E-03 2.3945072E-03 2.3916608E-03 2.2867515E-03 1.8710463E-03 1.3113051E-03 4.5449196E-04 4.5384822E-04 4.3013733E-04 3.3620227E-04 2.0972716E-04 1.6665788E-05 1.5978794E-05 1.4637693E-05 1.1350672E-05 7.4549748E-06 2.7311444E-06 1.0797340E-06 5.9363384E-07 3.9757629E-07 2.7179530E-07 1.6253658E-07 9.1042793E-08 4.8456979E-08 2.4876275E-08 1.2464740E-08 6.0717409E-09 2.9308100E-09 1.4089051E-09 6.7699735E-10 3.2603895E-10 1.6513572E-10 8.4049628E-11 4.3026464E-11 2.2167667E-11 1.1499778E-11 6.4303957E-12 3.6170782E-12 2.0468885E-12 1.1653908E-12 6.6757472E-13 4.1264198E-13 2.5624476E-13 1.5985729E-13 1.0018200E-13 6.3067830E-14 4.2522951E-14 2.8764500E-14 1.9520576E-14 1.3289678E-14 9.0762316E-15 6.5745639E-15 4.7730836E-15 3.4728887E-15 2.5323945E-15 1.8505868E-15 1.4207777E-15 1.0924514E-15 8.4126095E-16 6.4879145E-16 5.0109331E-16 4.0318874E-16 3.2474814E-16 2.6183612E-16 2.1132595E-16 1.7073111E-16 1.4267265E-16 1.1930970E-16 9.9842394E-17 8.3609597E-17 7.0064362E-17 6.0370506E-17 5.2043177E-17 4.4886152E-17 3.8731951E-17 3.3437542E-17 2.9536724E-17 2.6099836E-17 2.3070602E-17 2.0399783E-17 1.8044098E-17 1.6265762E-17 1.4666164E-17 1.3226948E-17 1.1931767E-17 1.0765883E-17 9.8675037E-18 9.0455633E-18 8.2934973E-18 7.6052086E-18 6.9752038E-18 6.4811958E-18 6.0228278E-18 5.5976218E-18 5.2029644E-18 4.8367794E-18 4.5452917E-18 4.2716910E-18 4.0149730E-18 3.7739983E-18 3.5477619E-18 3.3654062E-18 3.1925638E-18 3.0288322E-18 2.8736757E-18 2.7266255E-18 2.6067747E-18 2.4922832E-18 2.3828832E-18 2.2784407E-18 2.1786876E-18 2.0965540E-18 2.0176360E-18 1.9417327E-18 1.8687100E-18 1.7985010E-18 1.7402838E-18 1.6839425E-18 1.6294770E-18 1.5768203E-18 1.5258385E-18 1.4832977E-18 1.4418959E-18 1.4016999E-18 1.3626429E-18 1.3247247E-18 1.2928358E-18 1.2616840E-18 1.2312691E-18 1.2016581E-18 1.1727840E-18 1.1482644E-18 1.1243478E-18 1.1009002E-18 1.0779885E-18 1.0555457E-18 1.0364527E-18 1.0176946E-18 9.9927140E-19 9.8118324E-19 9.6349698E-19 9.4828957E-19 9.3341707E-19 9.1867858E-19 9.0427498E-19 8.9007240E-19 8.7781265E-19 8.6582086E-19 8.5396307E-19 8.4223923E-19 8.3071640E-19 7.4650584E-19 6.7107147E-19 6.0334130E-19 5.4264537E-19 4.8811287E-19,4.5017599E+02
```

Comment:
I have fixed this in the next poll request: https://github.com/gem/oq-risklib/pull/553